docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,468 @@
+
+![PortalRun](./media/portal_runs1.png)
+
+## Use CLI to view workflow tasks
+
+You can also run the following CLI show command to see more details on each task and the general workflow. The command will output
+- Schedule
+- Creation date
+- System data such as last modified date, by who, etc.
+
+Command Schema
+```sh
+az acr supply-chain workflow show -r <registry> -g <resourceGroup> -t continuouspatchv1   
+```
+Example Command 
+```sh
+az acr supply-chain workflow show -r myRegistry -g myResourceGroup -t continuouspatchv1 
+```
+Help command to see all required/optional flags
+```sh
+az acr supply-chain workflow show --help
+```
+
+## Updating the Continuous Patching Workflow
+
+In scenarios where you want to make edits to your continuous patching workflow, the update command is the easiest way to do so. You can update your schedule or JSON config schema with the update CLI command directly. 
+
+Command Schema
+```sh
+az acr supply-chain workflow update -r <registry> -g <resourceGroup> -t continuouspatchv1 --config <JSONfilename> --schedule <number of days>
+```
+Example Command 
+```sh
+az acr supply-chain workflow update -r myRegistry -g myResourceGroup -t continuouspatchv1 --config ./continuouspatching.json --schedule 1d
+```
+Help command to see all required/optional flags
+```sh
+az acr supply-chain workflow update --help
+```
+
+To update your schedule, run the previous command with a new input for schedule. To update your JSON configuration, we recommend making changes to the file, running a dry-run, and running the update command. 
+
+You can verify the updated workflow configuration by running the following show command or by clicking into your registry portal view. 
+```sh
+az acr supply-chain workflow show -r myregistry -g myresourcegroup -t continuouspatchv1
+```
+
+**NOTE:**
+Run Immediately: When Create and Update commands are executed, new Tasks will be queued based on the specified "schedule". This is behavior is intentional to avoid unnecessary patch runs for small workflow configuration updates that could incur expensive costs. If an immediate run is desired, the [--run-immediately] parameter may be provided.
+
+## Deleting the Continuous Patching Workflow
+
+To delete the continuous patching workflow, please run the following CLI command
+
+Command Schema
+```sh
+az acr supply-chain workflow delete -r <registry> -g <resourceGroup> -t continuouspatchv1 
+```
+Example Command 
+```sh
+az acr supply-chain workflow delete -r myregistry -g myresourcegroup –t continuouspatchv1
+```
+Help command to see all required/optional flags
+```sh
+az acr supply-chain workflow delete --help
+```
+
+Once a workflow is successfully deleted, the repository “csscpolicies/patchpolicy” will be automatically deleted. The 3 tasks that run your workflow will also be automatically deleted, along with any currently queued runs. 
+
+## Listing Running Tasks
+
+To list the most recently executed CSSC tasks, the following List command is available:
```

Comment:
CSSC is suddenly mentioned for the first time in this doc without explaining what it means. Please use another term like "Continuous Patching Tasks".